src/diverse_seq/distance.py

```diff
@@ -65,6 +68,10 @@ def __init__(
         if mash_canonical_kmers is None:
             mash_canonical_kmers = False
 
+        if distance_mode not in ("mash", "euclidean"):
+            msg = f"Unexpected distance {distance_mode}."
```

Comment:
change `f"Unexpected distance {distance_mode}."` to `f"Unexpected distance {distance_mode!r}."` Makes the error message easier to read by using the representation.